testCommon/src/test/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -36,8 +36,47 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     return stdout;
   }
 
+  @SneakyThrows
+  public static String runGitMacheteCommandAndReturnStdout(Path workingDirectory, int timeoutSeconds, String... arguments) {
+    String[] gitMachete = {"python", "-m", "git_machete.cmd"};
+    int sizeOfGitMachete = gitMachete.length;
+    int sizeOfArguments = arguments.length;
+    int sizeOfNewGitMachete = sizeOfGitMachete + sizeOfArguments;
+    gitMachete = Arrays.copyOf(gitMachete, sizeOfNewGitMachete);
+    for (int i = 0; i < sizeOfArguments; i++) {
```

Comment:
Nitpick, you can probably use `System.arraycopy` here: https://stackoverflow.com/a/2926621/2750812